### PR TITLE
BluecatAddressManager enhancement- fix exceptions issue

### PR DESCRIPTION
**Pull request by ohaim1008**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: demisto/etc#32401

## Description
The integration didn't catch all possible exceptions, therefor when a specific exception occurred it was raise from the integration and wasn't passed on to return_error.

Was checked with the new credentials, that are currntly only in centrify.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [x] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/BluecatAddressManager/ReleaseNotes/1_0_1.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Bluecat Address Manager
+Maintenance and stability enhancements.
```
Comment (bakatzir) [inline Packs/BluecatAddressManager/ReleaseNotes/1_0_1.md:4]:
```suggestion
- Maintenance and stability enhancements.
- Upgraded the Docker image to: *demisto/python3:3.9.1.14969*.
```

File: Packs/BluecatAddressManager/Integrations/BluecatAddressManager/BluecatAddressManager.py [hunk 2]
```diff
@@ -1,140 +1,136 @@
...
+        """
+        user_conf = self.user_conf
+        params = {
+            'type': 'Configuration',
+            'start': 0,
+            'count': 100
+        }
+        confs = self.http_request('GET', '/getEntities', params)
+        if not confs:
+            return_error('No configurations could be fetched from the system')
```
Comment (bakatzir) [inline Packs/BluecatAddressManager/Integrations/BluecatAddressManager/BluecatAddressManager.py:127]:
```suggestion
            raise Exception('No configurations could be fetched from the system.')
```

**Review by bakatzir — COMMENTED** [review]
Please show me that the TPB is green locally and  i will force merge.

File: Packs/BluecatAddressManager/Integrations/BluecatAddressManager/BluecatAddressManager.py (hunk 2 above)
Comment (ohaim1008) [reply Packs/BluecatAddressManager/Integrations/BluecatAddressManager/BluecatAddressManager.py:127]: will change the other return_errors too

**Review by bakatzir — APPROVED** [review]

**Pull request merged**